lib/screens/v2/widget_instance/shuttle_bus_info.ex

```diff
@@ -75,7 +75,10 @@ defmodule Screens.V2.WidgetInstance.ShuttleBusInfo do
   def audio_view(_instance), do: ScreensWeb.V2.Audio.ShuttleBusInfoView
 
   defp get_minute_range(schedule, now) do
-    {:ok, now} = DateTime.shift_zone(now, "America/Los_Angeles")
+    # Shift to find current service day. i.e. 1AM Monday is actually still in the Sunday service day.
+    {:ok, service_day_now} = DateTime.shift_zone(now, "America/Los_Angeles")
```

Comment:
Does LA/California observe DST the same way as Massachusetts? :thinking:

I'm wondering if `DateTime.add(now, some_number, :hour)` might be more appropriate / correct under DST nonsense.

Would you mind writing a few tests to make sure this function works as expected under normal circumstances as well as around a DST shift?